pkg/azuredisk/azuredisk_v2.go

```diff
@@ -110,60 +112,129 @@ type DriverV2 struct {
 	azDriverNodeInformer              azdiskinformertypes.AzDriverNodeInformer
 	deviceChecker                     *deviceChecker
 	kubeClientQPS                     int
+	isControllerPlugin                bool
+	isNodePlugin                      bool
 }
 
 // NewDriver creates a driver object.
-func NewDriver(options *DriverOptions) CSIDriver {
-	return newDriverV2(options, *driverObjectNamespace, *nodePartition, *controllerPartition, *heartbeatFrequencyInSec, *controllerLeaseDurationInSec, *controllerLeaseRenewDeadlineInSec, *controllerLeaseRetryPeriodInSec, *leaderElectionNamespace)
+func NewDriver(config *azdiskv1beta2.AzDiskDriverConfiguration) CSIDriver {
+	getDriverConfig(config)
+	return newDriverV2(config)
+}
+
+func getDriverConfig(driverConfig *azdiskv1beta2.AzDiskDriverConfiguration) {
+	if driverConfig.ObjectNamespace == "" {
```

Comment:
Rather than treat each deprecated parameter and its corresponding replacement config value separately, it would be fine to completely ignore the deprecated parameters if the `--config` parameter was specified.